### PR TITLE
Foundations - RPS Project: Remove recursion 

### LINKED ISSUE #26963: Foundations - RPS Project: Unclear instruction
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

This line is driving me nuts...
> Account for TIES by re-playing the round.
Is the intent here that learners should re-run the playRound function if a tie occurs (ie, in the game function), or are we asking them to recurse the playRound function when a tie occurs until a result is achieved?

Personally, I think what we really want is for them to return a string "You tied, try again!".  And then, in the game() function, we want them to implement logic to rerun the turn.

Either way, I think we need to reword this to be clear about what we want. 

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-rock-paper-scissors

### (Optional) Discord Name

_No response_

### (Optional) Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by fabulousgk**
## Because

The current instructions require some kind of recursion or looping which has not been taugt yet. It also uses a fucntion name not inline with recomended patterns.  THis PR corrects both.

## This PR

- Rename game() to playGame()
- Remove instruction to handle a tie by replaying the round
- Reword the instuction to play five total rounds instead of best of five.

## Issue

Closes #26963

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/javascript_basics/project_rock_paper_scissors.md [hunk 1]
```diff
@@ -19,7 +19,6 @@ Don't forget to commit early & often! You can [reference the Commit Message less
 3.  Your game is going to play against the computer, so begin with a function called `getComputerChoice` that will randomly return either 'Rock', 'Paper' or 'Scissors'.  We'll use this function in the game to make the computer's play. *Tip: use the console to make sure this is returning the expected output before moving to the next step!*
 4.  Write a function that plays a single round of Rock Paper Scissors.  The function should take two parameters - the `playerSelection` and `computerSelection` - and then return a string that declares the winner of the round like so: `"You Lose! Paper beats Rock"`
     *   Make your function's playerSelection parameter case-insensitive (so users can input `rock`, `ROCK`, `RocK` or any other variation).
-    *   Account for TIES by re-playing the round.
```
Comment (wise-king-sullyman) [inline foundations/javascript_basics/project_rock_paper_scissors.md:22]: I'm on the fence about just removing this. I feel like if we don't explicitly say what to do about ties we'll get _lots_ of questions about it.

Comment (fabulousgk) [reply foundations/javascript_basics/project_rock_paper_scissors.md:22]: Perhaps just a small change here:

> and then return a string that declares the winner or tie of the round like so: `"You Lose! Paper beats Rock"`

Comment (wise-king-sullyman) [reply foundations/javascript_basics/project_rock_paper_scissors.md:22]: Yeah that might be enough.

**Review by kamranzafar4343 — COMMENTED** [review]
"Thank you for your contribution! I appreciate the effort you've put into this commit. The code looks clean and well-organized. Keep up the good work!"

File: foundations/javascript_basics/project_rock_paper_scissors.md (hunk 1 above)
Comment (fabulousgk) [reply foundations/javascript_basics/project_rock_paper_scissors.md:22]: @wise-king-sullyman can you please review my addition when you have a chance.  Thank you !

**Review by wise-king-sullyman — APPROVED** [review]
Looks great, thanks!
